docs/manual/inference.tex

```diff
@@ -149,10 +149,21 @@
 to the last \<javac> command that was run. The annotation files generated in
 each round of inference appear in directories labeled \<iteration0>, \<iteration1>, etc.
 
+The \<wpi.sh> script is most useful when analyzing projects that follow the standard
+conventions of their build system for single-module projects. When analyzing a project
+that requires non-standard build system commands, use the \<-c> and \<-b> options to
+override the defaults used by \<wpi.sh>. For example, suppose that you wanted to use
+\<wpi.sh> to infer annotations for the Resource Leak Checker on
+\ahref{https://github.com/apache/zookeeper}{Apache Zookeeper's}
+\<zookeeper-server> module only. Without \<wpi.sh>, one might use the following command:
+\<mvn -B --projects zookeeper-server --also-make install -DskipTests>. To use this command via
+\<wpi.sh>, use the following:
+\<sh wpi.sh -b "-B --projects zookeeper-server --also-make" -c "install -DskipTests" -- resourceleak>.
```

Comment:
Would this have worked if `-DskipTests` were in the `-b` command-line argument rather than the `-c` command-line argument?

Reply:
Possibly? Skipping tests on a `clean` command doesn't make any sense, but I don't think Maven would throw an error because of it.

Reply:
I put it in the `-c` argument both because it makes more sense only for `install` and because I wanted to show that the user can put command-line arguments there, too, if they're only for the compilation phase.